apps/lockfile-explorer-web/src/containers/LockfileEntryDetailsView/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styles from './styles.scss';
+import appStyles from '../../appstyles.scss';
+import { IDependencyType, LockfileDependency } from '../../parsing/LockfileDependency';
+import { useAppDispatch, useAppSelector } from '../../store/hooks';
+import { pushToStack, selectCurrentEntry } from '../../store/slices/entrySlice';
+import { ReactNull } from '../../types/ReactNull';
+import { LockfileEntry } from '../../parsing/LockfileEntry';
+
+enum InfluencerTypes {
+  DETERMINANT,
+  TRANSITIVE_REFERRER
+}
+
+interface IInfluencerType {
+  entry: LockfileEntry;
+  type: InfluencerTypes;
+}
+
+export const LockfileEntryDetailsView = (): JSX.Element | ReactNull => {
+  const selectedEntry = useAppSelector(selectCurrentEntry);
+  const dispatch = useAppDispatch();
+
+  const [inspectDep, setInspectDep] = useState<LockfileDependency | null>(null);
+  const [influencers, setInfluencers] = useState<IInfluencerType[]>([]);
+
+  useEffect(() => {
+    if (selectedEntry) {
+      setInspectDep(null);
+    }
+  }, [selectedEntry]);
+
+  const selectResolvedEntry = useCallback(
+    (dependency) => () => {
+      if (inspectDep && inspectDep.name === dependency.name) {
+        if (dependency.resolvedEntry) {
+          dispatch(pushToStack(dependency.resolvedEntry));
+        } else {
+          console.error('No resolved entry for dependency: ', dependency);
+        }
+      } else if (selectedEntry) {
+        setInspectDep(dependency);
+        // calculate influencers
+        const stack = [selectedEntry];
+        const determinants = new Set<LockfileEntry>();
+        const transitiveReferrers = new Set<LockfileEntry>();
+        const visitedNodes = new Set<LockfileEntry>();
+        visitedNodes.add(selectedEntry);
+        while (stack.length) {
+          const currEntry = stack.pop();
+          if (currEntry) {
+            for (const referrer of currEntry.referencers) {
+              let flag = false;
```

Comment:
Find a more meaningful name for this variable.  Maybe `hasDependency`?

Reply:
changed